spec/motion-spec/matchers/change_spec.rb

```diff
@@ -0,0 +1,62 @@
+# -*- encoding : utf-8 -*-
+describe 'Matcher::Change' do
+  class TestClass
+    attr_reader :counter
+    def initialize
+      @counter = 0
+    end
+
+    def add
+      @counter += 1
+    end
+
+    def dont_add; end
+  end
+
+  before { @test_object = TestClass.new }
+
+  context 'when the expected block changes the argument block' do
+    it 'passes' do
+      expect { @test_object.add }.to change { @test_object.counter }
+    end
+  end
+
+  context "when the expected block doesn't change the argument block" do
+    it "fails" do
+      expect_failure('Block expected to change value') do
+        expect { @test_object.dont_add }.to change { @test_object.counter }
+      end
+    end
+
+    it "passes when asked not_to" do
+      expect { @test_object.dont_add }.not_to change { @test_object.counter }
+    end
+  end
+
+  context "when specified 'by'" do
+    it "passes when changes are by the given amount" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.